.github/workflows/jobs.yml

```diff
@@ -261,19 +267,26 @@ jobs:
         uses: msys2/setup-msys2@v2
         with:
           update: true
-          msystem: MINGW64
+          msystem: ${{ matrix.msystem }}
           install: >-
             base-devel
             git
-            mingw-w64-x86_64-cmake
-            mingw-w64-x86_64-ninja
-            mingw-w64-x86_64-gcc-fortran
-            mingw-w64-x86_64-openblas
-            mingw-w64-x86_64-msmpi
+          pacboy: >-
```

Comment:
What's pacboy? Can you detail?

Reply:
https://www.msys2.org/docs/package-naming/#avoiding-writing-long-package-names